src/build/util.ts

```diff
@@ -22,3 +25,8 @@ export class Logger extends Transform {
     callback(null, file);
   }
 }
+
+export function optimizePipe(stream: stream.Stream) {
```

Comment:
util.ts seems like an odd locations for this, since it defines part of the build pipeline